### PR TITLE
Removes from existence once and for all the experimentor

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

As the title say, this PR just removes the experimentor and all of it's references (save for the .dmi).

## Why It's Good For The Game

Aside being in the way of the more recent anomalies system, it's a really, and I mean, REALLY out of date code and mechanic, it's effects in the wrong hands could cause a lot of troubles, thankfully due to how out-of-date and convoluted the system it's almost impossible to do so.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

See MapDiffBot

</details>

## Changelog
:cl:
del: Decomissioned for good the E.X.P.E.R.I-MENTOR in favour of the anomaly system.
add: All of the Anomaly research room has been refurbished properly.
/:cl:

### CODE REVIEW

**Comment by MarkusLarsson421** [comment]
No more strange objects all over the place?

**Comment by burn1cycle** [comment]
I agree with Mark's point. It takes up a random item spawn slot that could be used for... Empty syndicakes boxes or crushed soda cans. OK, so our random item spawns kinda suck but they would suck worse with items for a outdated system because then we have to explain to new players "there was a hole here. its gone now".

**Comment by itsmeow** [comment]
Yeah this system kinda sucks and xenoarch has almost entirely replaced it.

Should we also remove the destructive analyzer? I have yet to figure out how it works.

**Comment by MarkusLarsson421** [comment]
> Should we also remove the destructive analyzer? I have yet to figure out how it works.

That's used for getting illegal and alien tech, not much else.

**Comment by Rukofamicom** [comment]
fix this up and ping if you're still interested in removing this.

**Comment by Rukofamicom** [comment]
Removing label per ping in discord

**Review by DrDuckedGoose — APPROVED** [review]

**Review by Rukofamicom — APPROVED** [review]

File: _maps/RandomZLevels/snowdin.dmm [hunk 1]
```diff
@@ -194,7 +194,6 @@
 "aQ" = (
 /obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{
 	dir = 6;
-	piping_layer = 3;
```
Comment (EvilDragonfiend) [inline _maps/RandomZLevels/snowdin.dmm:197]: why are those piping layers are randomly removed? are those fine?

**Comment by Rukofamicom** [comment]
Due to the nature of the conflicts, I give permission to merge this in my name the next time it is deconflicted. 

I know you pinged me earlier today while I was at work and it was conflicted again by the time I got home lol

**Comment by EvilDragonfiend** [comment]
I think we can just let xenoarchaenology PR handle this. They're removing experimentor too.

**Comment by Rukofamicom** [comment]
Who keeps merging map things at the same time every day?!

**Comment by Tsar-Salat** [comment]
> Who keeps merging map things at the same time every day?!

Ironically, probably Pigeon

**Comment by MarkusLarsson421** [comment]
> Ironically, probably Pigeon 

This time, it actually was. lol

My minor map changes was merged by him.

**Pull request merged**